data/reference/README.md

```diff
@@ -0,0 +1,5 @@
+# Reference directory files and origins
+
+- `FLAGS.tsv`:
+The FLAGS.tsv file is a list of commonly mutated genes in exome sequencing samples, derived from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4267152/.
+This file contains the top 50 genes from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706417/bin/12920_2017_309_MOESM3_ESM.txt
```

Comment:
I think it might be worthwhile to create this file with a shell script - `curl` and `head -n 50 > FLAGS.tsv`. It would jump off very nicely from the fastq activity as a concrete example.

Reply:
I was debating this, so I'll go ahead and do it. I honestly didn't remember how much processing was involved, but not much.

Reply:
Script looks good!